src/integrationTest/java/uk/gov/hmcts/reform/wataskmanagementapi/services/calendar/PublicHolidaysCollectionTest.java

```diff
@@ -21,16 +23,30 @@ class PublicHolidaysCollectionTest {
 
     @Test
     public void should_not_return_empty_bank_holidays() {
-        Set<LocalDate> response = publicHolidaysCollection.getPublicHolidays(CALENDAR_URI);
+        Set<LocalDate> response = publicHolidaysCollection.getPublicHolidays(List.of(CALENDAR_URI));
         assertThat(response).isNotEmpty();
     }
 
     @Test
     public void should_call_external_api_only_once() {
-        Set<LocalDate> resultFromApi = publicHolidaysCollection.getPublicHolidays(CALENDAR_URI);
-        Set<LocalDate> resultFromCache = publicHolidaysCollection.getPublicHolidays(CALENDAR_URI);
-        Set<LocalDate> resultFromCacheAgain = publicHolidaysCollection.getPublicHolidays(CALENDAR_URI);
+        BankHolidays resultFromApi = publicHolidaysCollection.getPublicHolidays(CALENDAR_URI);
+        BankHolidays resultFromCache = publicHolidaysCollection.getPublicHolidays(CALENDAR_URI);
+        BankHolidays resultFromCacheAgain = publicHolidaysCollection.getPublicHolidays(CALENDAR_URI);
 
         assertThat(resultFromApi).isSameAs(resultFromCache).isSameAs(resultFromCacheAgain);
     }
+
+    @Test
+    public void secondCalendarShouldOverrideMainCalendar() {
+        List<String> oneUri = List.of(CALENDAR_URI, "https://raw.githubusercontent.com/hmcts/wa-task-management-api/RWA-1768-calendar-for-test/src/test/resources/override-working-day-calendar.json");
+        Set<LocalDate> oneCalendarResult = publicHolidaysCollection.getPublicHolidays(oneUri);
+        assertThat(oneCalendarResult.contains(LocalDate.of(2022, 12, 26)));
+        assertThat(oneCalendarResult.contains(LocalDate.of(2022, 12, 27)));
+
+        List<String> twoUris = List.of(CALENDAR_URI, "https://raw.githubusercontent.com/hmcts/wa-task-management-api/RWA-1768-calendar-for-test/src/test/resources/override-working-day-calendar.json");
+        Set<LocalDate> fullResult = publicHolidaysCollection.getPublicHolidays(twoUris);
+        assertThat(!oneCalendarResult.contains(LocalDate.of(2022, 12, 26)));
```

Comment:
`fullResult` not used and could not understand how `oneCalendarResult` changed :)